packages/fa-icon-chooser/src/components/fa-icon-chooser/fa-icon-chooser.tsx

```diff
@@ -202,7 +202,21 @@ export class FaIconChooser {
   }
 
   getFamilies(): string[] {
-    return Object.keys(this.familyStyles);
+    // If we have kit metadata with embedProSvg permits, use those families
+    const embedProSvg = get(this, 'kitMetadata.permits.embedProSvg', []);
+
+    if (embedProSvg.length > 0) {
+      // Extract unique families from embedProSvg permits and sort alphabetically
+      const families = [...new Set(embedProSvg.map(permit => permit.family).filter(family => typeof family === 'string'))] as string[];
+      return families.sort();
+    }
+
+    // For version-only configurations (no kit), use families from familyStyles
+    if (this.familyStyles && typeof this.familyStyles === 'object') {
+      return Object.keys(this.familyStyles).sort();
+    }
```

Comment:
This `if` block is used for the free, pro lite and pro plus lite users when they update the `local.json` to something like:
```
{
  "props": {
    "version": "7.0.0-rc3"
  }
}
```

The `familyStyles` from [here](https://github.com/FortAwesome/fa-icon-chooser/pull/61/files#diff-b51e9f7ddaa940d0cc0d1c3ba5ba203b636535fd535c4e1aae553a8f0851823aL160) will be used.